src/persistence/recordings.ts

```diff
@@ -42,12 +17,8 @@ export function load(recording: string): MoujiRecording {
   return tracks;
 }
 
-export function save(recording: MoujiRecording): string {
-  if (!fs.existsSync(config.recordPath)) {
-    fs.mkdirSync(config.recordPath);
-  }
-
-  const file = path.join(config.recordPath, `${Date.now()}.moujirec`);
+export function save(recording: MoujiRecording, name: string): string {
+  const file = path.join(process.cwd(), `${name}.moujirec`);
```

Comment:
If we are allowed to specify a path to save, we don't have to make the specified path strictly inside the cwd or always have a file extension

We can definitely use this up a level if there's no path specified, but rn if I created a state dir at `/var/lib/mouji` and pass `-f /var/lib/mouji/boof`, it'll be interpreted as `${cwd}/var/lib/mouji/boof.mouji`

Might also consider adding some fs detection that appends a filename when a dir is specified for `-f` or smth but that's just extra

Might also consider creating the dir to which you would do the write if it doesn't already exist but that's extra